includes/Infrastructure/Helper/UrlHelper.php

```diff
@@ -13,10 +13,10 @@ class UrlHelper {
 	 *
 	 * @return string|null
 	 */
-	public static function checkAndCleanUrl( string $url, array $protocols = null, string $_context = 'display' ): ?string {
+	public static function checkAndCleanUrl( string $url, ?array $protocols = null, string $_context = 'display' ): ?string {
```

Comment:
?array $protocols = null 
array $protocols = [] possible ?

Reply:
we keep the default value of the proxified method